src/strings/PowerFxResources.en-US.resx

```diff
@@ -4522,6 +4522,10 @@
     <value>Ensure you have text for your identifier. This error occurs when the identifier is all blanks or spaces.</value>
     <comment>1 How to fix the error. </comment>
   </data>
+  <data name="ErrorResource_ErrUnOrderedTypeForComparison_Type_ShortMessage" xml:space="preserve">
```

Comment:
>ErrorResource_ErrUnOrderedTypeForComparison_Type_ShortMessage [](http://example.com/codeflow?start=14&length=61)

Since this resource doesn't have all of the short/long/howtofix, do we really need to define it using it? Can we define it as a "regular" / "stand-alone" resource?